[All Hosts] (devx) simplified scripts in project templates (#4923)
OfficeDev/office-js-docs-pr: docs/excel/custom-functions-debugging.md

You can use the following tasks to start debugging on desktop or online.

> - `npm run start:desktop`: Starts Excel on desktop and sideloads your add-in. If the "start:desktop" script is not present in the "scripts" section of the project's package.json file, then run `npm run start` instead.

Reviewer: ```suggestion
- `npm run start:desktop`: Starts Excel on desktop and sideloads your add-in. If the "start:desktop" script isn't present in the "scripts" section of the project's package.json file, then run `npm run start` instead.
```
